### ISSUE TITLE
Viral Robot polluted the whole issue page!!!

### ISSUE THREAD

**Issue by Jingnan-Jia** [open-mmlab/mmdetection#12250]
Can anyone help to remove the unrelated issues which were automatically proposed by viral robots.

**Comment by gboeer** [comment]
Issues you mean?

Did not see that happen often at GH ... but sadly it shows that the maintainers of this repo are gone.

**Comment by Jingnan-Jia** [comment]
> Issues you mean?
> 
> Did not see that happen often at GH ... but sadly it shows that the maintainers of this repo are gone.

yes. It seems that the maintainers from mmlab switched their direction to LLM. So I guess they don't care about mmdet any more.
